File: Iv Layhort/Java_Layhort/src/Task_3/Exercise_2_HashSet.java
package Task_3;
import java.util.*;

public class Exercise_2_HashSet {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        Set<String> mySet = new HashSet<String>();

        System.out.println("Enter the size :");
        int size = input.nextInt();

        for(int i=0;i<size;i++){
            System.out.println("Enter the string pairs");
            String left = input.next();
            String right = input.next();

            mySet.add(left.toLowerCase() +" , "+ right.toLowerCase());

            System.out.println(mySet.size());
        }
    }
}

File: Iv Layhort/Scala-Layhort/src/Task_3/Exercise_2_HashSet.scala
package Task_3

import scala.collection.mutable
import scala.io.StdIn._
object Exercise_2_HashSet {
  def main(args: Array[String]): Unit = {
    var mySet:mutable.HashSet[String] = new mutable.HashSet[String]()

    println("Enter the size : ")
    val size = readInt()

    for(i <- 0 until size){
      println("Enter the string pairs ;)")
      val left = readLine()
      val right = readLine()

      mySet += (left+" , "+right)

      println(mySet.size)
    }
  }
}
